platform/darwin/src/MGLGeometry.h

```diff
@@ -14,6 +14,11 @@ typedef struct __attribute__((objc_boxable)) MGLCoordinateSpan {
     CLLocationDegrees longitudeDelta;
 } MGLCoordinateSpan;
 
+/* Defines a point on the map in Mercator projection for a specific zoom level. */
+typedef struct __attribute__((objc_boxable)) MGLMapPoint {
+    double x, y, zoom;
```

Comment:
Use `CGFloat` (a typedef of `double`) for `x` and `y`, for consistency with `CGPoint` etc. Rename `zoom` to `zoomLevel`, for consistency with the rest of the API.

Declarations of multiple fields should always be on separate lines and (for a public API) should be documented separately. Otherwise they don’t show up in the generated documentation, even if the containing type does.